boat-quay/boat-quay-rules/src/main/kotlin/com/backbase/oss/boat/quay/ruleset/PrefixPathRule.kt

```diff
@@ -0,0 +1,35 @@
+package com.backbase.oss.boat.quay.ruleset
+
+import com.typesafe.config.Config
+import org.zalando.zally.rule.api.*
+
+@Rule(
+        ruleSet = BoatRuleSet::class,
+        id = "B007",
+        severity = Severity.MUST,
+        title = "Check prefix for paths"
+)
+class PrefixPathRule(config: Config) {
+
+    private val validPathPrefixes = config
+            .getStringList("PrefixPathRule.validPathPrefixes")
+            .toSet()
+
+    @Check(Severity.MUST)
+    fun validate(context: Context): List<Violation>  =
+
+        context.api.paths.orEmpty()
+                .map {
+                    val extractParts = it.key.split("/")
+                    val prefix = if(extractParts.size  > 1) extractParts[1] else ""
+                    Pair(prefix, it.value)
+                }
+                .filter {
+                    !validPathPrefixes.contains(it.first)
+                }
+                .map {
+                    context.violation("Incorrect path prefix: ${it.first}", it.second)
```

Comment:
It would be good to inform what the correct values are. So in the violations we have info on what is wrong and then how to fix it.

Reply:
done